Resources/Prototypes/_Harmony/Catalog/uplink_catalog.yml

```diff
@@ -0,0 +1,25 @@
+# Guns
+# Explosives
+# Ammo
+# Chemicals
+# Deception
+
+- type: listing
+  id: UplinkSyndicateID
```

Comment:
```
- type: listing
  id: UplinkSyndicateIDHarmony
...
```

To avoid duplicate ID and instant test fail, in case this ever gets added upstream. Since it's the most likely ID to use for it.